### DISCUSSION TITLE
Sound null safety

### DISCUSSION THREAD

**Discussion by yom-ko** [sidrao2006/dynamic_cached_fonts#120] [Q&A] [answered]
Hi! Your package looks good! Are you planning to add support for [sound null safety](https://dart.dev/null-safety) in the near future?

**Comment by sidrao2006** [comment, marked as answer]
Hello and welcome! The main branch of the repo is null safe right now.. I'm planning to release it soon!

**Reply by yom-ko** [reply]
Awesome! Looking forward to the release. This package works like a charm for me so far. Good luck with your work!

**Reply by sidrao2006** [reply]
Thank you :)
v0.3.0 is now live! (with null safety 🎊)
https://pub.dev/packages/dynamic_cached_fonts/versions/0.3.0
